resources/views/site/contact.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Form\Widget\Form;
+use Yiisoft\Html\Html;
+
+?>
+
+<?= Html::beginTag('div', ['class' => 'column is-4 is-offset-4']) ?>
+
+    <?= Html::beginTag('p', ['class' => 'subtitle has-text-black']) ?>
+        Please fill out the following to Contact.
+    <?= Html::endTag('p') ?>
+
+    <?= Form::begin()
+        ->action($url->generate('site/contact'))
```

Comment:
`$url` should be annotated at the top of the file.

Reply:
done.